builders/ethereum/dev-env/thirdweb.md

```diff
@@ -134,6 +134,33 @@ Or install it into your existing project by running:
 npx thirdweb install
 ```
 
+### Specify Client ID {: #specify-client-id }
+
+Before you launch your dApp (locally or publicly deployed), you must have a ThirdWeb Client ID associated with your project. A ThirdWeb Client ID is synonymous with an API key. You can create a free API key by [signing into your ThirdWeb Account and navigating to Settings -> API Keys](https://thirdweb.com/create-api-key){target=\_blank}.
+
+Press **Create API Key** then take the following steps:
+
+1. Give your API key a name
+2. Enter the allowed domains that the API key should accept requests from. It's recommended that you allow only necessary domains, but for development purposes, you can select **Allow all domains**.
+3. Press **Next** and Confirm the prompt on the next page. 
+
+Finally, specify your Client ID (API Key) in your `client.ts` file. ThirdWeb by default assumes it will be in your .env file named `NEXT_PUBLIC_TEMPLATE_CLIENT_ID`.
```

Comment:
```suggestion
Finally, specify your Client ID (API Key) in your `client.ts` file. ThirdWeb by default assumes it will be in your `.env` file named `NEXT_PUBLIC_TEMPLATE_CLIENT_ID`.
```